studio/components/interfaces/Auth/AuthProvidersForm/FormField.tsx

```diff
@@ -61,6 +61,49 @@ const FormField: FC<Props> = ({ name, properties, formValues, disabled = false }
           }
         />
       )
+    case 'multiline-string':
```

Comment:
This is just a copy/paste of the `case 'string'` above, but returning `Input.TextArea` instead of `Input`

Reply:
shall we just use textarea to make it a bit simpler?

(sorry for mega late reply)